lumigator/backend/backend/tests/unit/services/test_workflow_service.py

```diff
@@ -0,0 +1,84 @@
+from uuid import UUID
+
+import pytest
+from lumigator_schemas.tasks import SYSTEM_PROMPT_DEFAULTS
+from lumigator_schemas.workflows import WorkflowCreateRequest
+
+from backend.tests.conftest import TEST_SEQ2SEQ_MODEL
+
+
+def test_workflow_request_requires_system_prompt_for_text_generation():
+    # Create a text generation task definition
+    task_definition = {"task": "text-generation"}
+
+    # Test that creating a WorkflowCreateRequest without system_prompt raises an error
+    with pytest.raises(ValueError) as excinfo:
+        WorkflowCreateRequest(
+            name="test_text_generation_run",
+            description="Test missing system prompt for text generation",
+            model="microsoft/Phi-3-mini-instruct",
+            provider="hf",
+            task_definition=task_definition,
+            # system_prompt is intentionally not provided
+            dataset=UUID("d34dd34d-d34d-d34d-d34d-d34dd34dd34d"),
+        )
+
+    # Verify exact error message from Pydantic validation
+    assert "Default system_prompt not available for text-generation" in str(excinfo.value)
+
+
+def test_default_system_prompt_for_summarization():
+    # Create a WorkflowCreateRequest without specifying system_prompt
+    workflow_request = WorkflowCreateRequest(
+        name="test_summarization_run",
+        description="Test default system prompt for summarization",
+        model=TEST_SEQ2SEQ_MODEL,
+        provider="hf",
+        task_definition={"task": "summarization"},
+        # system_prompt is intentionally not provided
+        dataset=UUID("d34dd34d-d34d-d34d-d34d-d34dd34dd34d"),
+    )
+
+    # Verify that the default system prompt for summarization was applied
+    expected_prompt = SYSTEM_PROMPT_DEFAULTS["summarization"]
+    assert workflow_request.system_prompt == expected_prompt
```

Comment:
whats the difference between this test and the previous one? 🤔  why does it set the fallback prompt here, but raises an error in the previous one? is the behavior different depending on the `task_definition`?